examples/checkin.py

```diff
@@ -0,0 +1,201 @@
+from hackerforms import *
+import requests, json
+from ast import literal_eval
+import os
+
+api_key = os.environ['AIRTABLE_API_KEY']
+
+fname = read("Welcome to Dr.Pete's! Let's get started. What is your first name?")
+
+lname = read('What is your last name?')
+
+minitial = read('What is your middle initial? If you have no middle initial, leave blank.')
+
+name = fname+' '+minitial+' '+lname
+
+email = read("Ok. What is your email?")
+
+birthdate = read_date(f"What is your date of birth, {fname}?")
+
+#get countries from api and populate dropdown
+list = []
+hed={}
+data = {}
+url = 'https://countriesnow.space/api/v0.1/countries'
+response = requests.get(url, json=data, headers=hed)
+response = response.json()
+list = response['data']
+
+countries = []
+for item in list:
+  if 'country' in item:
+    countries.append(item['country'])
+country = read_dropdown(
+  "In which country do you currently live?",
+  options = countries
+  )
+
+#get states by country and populate dropdown
+cities = []
+hed={"Content-Type": "application/json"}
+data = {"country": country}
+url = 'https://countriesnow.space/api/v0.1/countries/cities'
+response = requests.post(url, json=data, headers=hed)
+response = response.json()
+cities = response['data']
+
+city = read_dropdown(
+  "In which city do you currently live?",
+  options = cities
+  )
```

Comment:
Acho que rola simplificar esse também, fazendo um request só

```suggestion
countries = []
for item in list:
  countries.append({"label": item['country'], "value": item})

chosen_country = read_dropdown(
  "In which country do you currently live?",
  options = countries
  )

country = chosen_country["country"]

city = read_dropdown(
  "In which city do you currently live?",
  options = chosen_country["cities"]
  )
```